### ISSUE TITLE
minor change for better code implementation

### ISSUE THREAD

**Issue by akansh12** [explainingai-code/GANs-Pytorch#2]
- Maybe you can push [VAE-Pytorch](https://github.com/explainingai-code/VAE-Pytorch/tree/main)/[utils](https://github.com/explainingai-code/VAE-Pytorch/tree/main/utils/extract_mnist_images.py) to this repo too.
- ModuleNotFoundError: No module named 'dataset' pops up. It happened in VAE code too. So to include 
`import sys 
sys.path.append("./")
`

**Comment by explainingai-code** [comment]
When do you see the 'ModuleNotFoundError' ?

**Comment by akansh12** [comment]
while running
`python tools/train_gan.py`

**Comment by explainingai-code** [comment]
Yeah.. I suspected that. It should be run via `python -m tools.train_gan` which ensures the import correctly and not `python tools/train_gan.py` .  Same for VAE .
https://github.com/explainingai-code/GANs-Pytorch?tab=readme-ov-file#quickstart
